services/graph/pkg/service/v0/driveitems.go

```diff
@@ -211,6 +213,13 @@ func cs3ResourceToRemoteItem(res *storageprovider.ResourceInfo) (*libregraph.Rem
 	if res.Type == storageprovider.ResourceType_RESOURCE_TYPE_CONTAINER {
 		remoteItem.Folder = &libregraph.Folder{}
 	}
+	if res.GetSpace() != nil {
+		remoteItem.RootId = libregraph.PtrString(storagespace.FormatResourceID(*res.GetSpace().GetRoot()))
```

Comment:
This also. Not sure if the case `space != nil && root == nil` can happen but we should be prepared of it